src/i18n/fi/translation.json

```diff
@@ -204,7 +201,8 @@
   "privacy-policy": "Tietosuojaseloste",
   "profile": {
     "competences": {
-      "add": "Lisää osaaminen",
+      "add": "Lisää osaamisia",
+      "edit": "Muokkaa osaamisia",
```

Comment:
Good job of unifying these add/edit translations as they were unwanted duplication. 🚀